### DISCUSSION TITLE
Support lifetime of Qidi Slicer

### DISCUSSION THREAD

**Discussion by TheMechaniac** [QIDITECH/QIDISlicer#102] [Q&A]
With Qidi Slicer and Qidi Studio existing simultaneously, what is the end-of-life plan for Qidi Slicer? The flatter single-window GUI of Slicer makes changing Advanced settings faster and easier than Studio, and so I prefer Slicer. I'd like to know what I need to plan to convert/maintain profiles in software going forward. Thanks!

**Comment by QIDITECH** [comment]
We will maintain both software. slicer has not stopped updating, but because we are in the middle of a big update, so we need more time to test, and soon we will update slicer.
